lists/global.csv

```diff
@@ -1505,4 +1505,13 @@ https://www.facebook.com/RTNews,GRP,Social Networking,2022-03-06,OONI,
 https://www.ideasbeyondborders.org/,CULTR,Culture,2022-03-16,OONI,
 https://www.starlink.com/,ANON,Anonymization and circumvention tools,2022-03-30,community member,Censorship circumvention and emergency internet access
 https://dailystormer.name/,POLR,Political Criticism,2022-03-18,community member,American far-right website; supposedly blocked in Europe
-https://www.microsoft.com/,COMT,Communication Tools,2022-03-30,OONI,Reportedly blocked in Tunisia but worth testing globally
\ No newline at end of file
+https://www.microsoft.com/,COMT,Communication Tools,2022-03-30,OONI,Reportedly blocked in Tunisia but worth testing globally
+https://teams.microsoft.com/,COMT,Communication Tools,2022-03-31,OONI,
+https://login.microsoftonline.com/common/v2.0/.well-known/openid-configuration,COMT,Communication Tools,2022-03-31,OONI,
+https://statics.teams.cdn.office.net/hashed/favicon/prod/favicon-9e2b8f1.ico,COMT,Communication Tools,2022-03-31,OONI,
+https://login.live.com/Me.htm?v=3,COMT,Communication Tools,2022-03-31,OONI,
+https://aadcdn.msauth.net/shared/1.0/content/images/favicon_a_eupayfgghqiai7k9sol6lg2.ico,COMT,Communication Tools,2022-03-31,OONI,
```

Comment:
Based on the PCAPs I have seen of the microsoft teams app, the only domain used by the app is https://teams.microsoft.com/.

I suggest dropping all other URLs and keeping just that.

Reply:
Required for use on the web. (login)

(PCAPs were probably created after login or in use. In this case, some important and basic addresses may not be visible.)